packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -297,5 +309,85 @@ export const SlackNotifier: Notifier = {
 
   async integrationUpdated() {
     // Slack sends a system message on its own
+  },
+
+  async shareTopic(
+    dataLoader: DataLoaderWorker,
+    userId: string,
+    teamId: string,
+    meetingId: string,
+    reflectionGroupId: string,
+    stageIndex: number,
+    channelId: string
+  ) {
+    const r = await getRethink()
+    const [user, team, meeting, reflectionGroup, reflections, slackAuth] = await Promise.all([
+      dataLoader.get('users').load(userId),
+      dataLoader.get('teams').load(teamId),
+      dataLoader.get('newMeetings').load(meetingId),
+      dataLoader.get('retroReflectionGroups').load(reflectionGroupId),
+      r.table('RetroReflection').getAll(reflectionGroupId, {index: 'reflectionGroupId'}).run(),
+      r.table('SlackAuth').getAll(userId, {index: 'userId'}).filter({teamId})(0).run()
```

Comment:
-1 if there's no slackAuth, I think this would cause an error. Safer to do: `.nth(0).default(null)`